### PR TITLE
Add border in images, footer and responsive design

### PR DESCRIPTION
**Pull request by Jomalagon**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -10,7 +13,11 @@
     } 
     .img-portada {
         block-size: 23vh;
+        
     }
+    .imago{
+    block-size:15vh;
```
Comment (axelzo) [inline style.css]: 💅 I think you could rename this class with something more descriptive and try to follow BEM nomenclature.
Also you have wrong indentation.

File: style.css [hunk 2]
```diff
@@ -62,10 +69,56 @@
     justify-content: center;
     text-align: center;
     margin-top: 3rem;
+
```
Comment (axelzo) [inline style.css]: 💅  Remove this empty line please.

File: style.css [hunk 3]
```diff
@@ -1,3 +1,6 @@
+body {
+    margin: auto 0;
+}
 .navigation {
```
Comment (axelzo) [inline style.css]: 💅 Remember to have empty line after class curly bracket.

File: style.css [hunk 4]
```diff
@@ -62,10 +69,56 @@
...
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 20px;
+    font-size: 15px;
+    color: black;
+}
+
+.img-portada {
```
Comment (axelzo) [inline style.css]: Same, try to have more descriptives names.

**Review by axelzo — CHANGES_REQUESTED** [review]

**Comment by axelzo** [comment]
@Jomalagon So far yo did a great job, i left some comments.

File: style.css (hunk 1 above)
Comment (Jomalagon) [reply style.css]: ready

File: style.css (hunk 2 above)
Comment (Jomalagon) [reply style.css]: Ready

File: style.css (hunk 4 above)
Comment (Jomalagon) [reply style.css]: And ready

**Comment by Jomalagon** [comment]
@axelzo i already addressed your comments. it is ready for another review

**Comment by axelzo** [comment]
@Jomalagon Amazing job, It looks great! Thank you so much. Approved.

**Comment by Jomalagon** [comment]
@axelzo thanks to you!

**Pull request merged**